OSSM 3.0 TP1: OSSM-6060 OSSM3 with cert-manager and istio-csr (#83665)
openshift/openshift-docs: modules/ossm-cert-manager-istio-csr-inplace-update-strategy.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc

:_mod-docs-content-type: PROCEDURE
[id="inplace-istio-csr-installation_{context}"]
= istio-csr installation: InPlace update strategy

> The default for `spec.updateStrategy` for the `Istio` resource is `InPlace.` Follow this procedure if you going to leave `spec.updateStrategy` as `InPlace` when you create and install your `Istio` resource.

Reviewer: ```suggestion
For the `Istio` resource, the default value for the `spec.updateStrategy` field is `InPlace.` Follow this procedure if you plan to leave `spec.updateStrategy` as `InPlace` when you create and install your `Istio` resource.
```

.Procedure

> . Add the jetstack charts to your local Helm repo:

Reviewer: ```suggestion
. Add the jetstack charts to your local Helm repo by running the following command:
```
For accessibility purposes, it's helpful to include the text "by running the following command" to help orient people who are using tools like screen readers to access this content.

> . Add the jetstack charts to your local Helm repo by running the following command:

Reviewer 2: ```suggestion
. Add the Jetstack charts repository to your local Helm repository by running the following command:
```

+
[source, terminal]
----
helm repo add jetstack https://charts.jetstack.io --force-update
----

> . Install the `istio-csr`chart:

Reviewer: ```suggestion
. Install the `istio-csr` chart by running the following command:
```

> helm repo add jetstack https://charts.jetstack.io --force-update

Reviewer 2: Terminal command prompts (`$`) throughout.
